packages/frontend-v2/pages/index.tsx

```diff
@@ -290,137 +316,138 @@ const Home: NextPage = () => {
       <br />
       <div>
         <h2>API Error Handling</h2>
         <div>
           <h3>Toasts without logging</h3>
-          <button
+          <Button
             onClick={() =>
               toast.info("Oh btw here's some info on what you were doing")
             }
           >
             info
-          </button>
-          <button
+          </Button>
+          <Button
             onClick={() =>
               toast.success("Whatever you were doing was successful")
             }
           >
             success
-          </button>
-          <button
+          </Button>
+          <Button
             onClick={() =>
               toast.warn("Whatever you were doing was kinda successful")
             }
           >
             warning
-          </button>
-          <button
+          </Button>
+          <Button
             onClick={() => toast.error("Whatever you were doing failed lol")}
           >
             error
-          </button>
+          </Button>
         </div>
       </div>
       <div>
         <h3>Toasts with logging</h3>
-        <button
+        <Button
           onClick={() =>
             toast.info("Oh btw here's some info on what you were doing", {
               log: true,
             })
           }
         >
           info
-        </button>
-        <button
+        </Button>
+        <Button
           onClick={() =>
             toast.success("Whatever you were doing was successful", {
               log: true,
             })
           }
         >
           success
-        </button>
-        <button
+        </Button>
+        <Button
           onClick={() =>
             toast.warn("Whatever you were doing was kinda successful", {
               log: true,
             })
           }
         >
           warning
-        </button>
-        <button
+        </Button>
+        <Button
           onClick={() =>
             toast.error("Whatever you were doing failed lol", { log: true })
           }
         >
           error
-        </button>
+        </Button>
       </div>
       <div>
         <h2>Client Side Error Handling</h2>
-        <button
+        <Button
           onClick={() => {
             setIsClientSideError(true);
           }}
         >
           Trigger a client side error
-        </button>
+        </Button>
       </div>
       <div>
         <h2>Logging</h2>
-        <button onClick={() => logger.info("Info log")}>info</button>
-        <button onClick={() => logger.debug("Debug log")}>debug</button>
-        <button onClick={() => logger.warn("Warning log")}>warning</button>
-        <button onClick={() => logger.error("Error log")}>error</button>
+        <Button onClick={() => logger.info("Info log")}>info</Button>
+        <Button onClick={() => logger.debug("Debug log")}>debug</Button>
+        <Button onClick={() => logger.warn("Warning log")}>warning</Button>
+        <Button onClick={() => logger.error("Error log")}>error</Button>
```

Comment:
in the future, no need to make button changes, since this is temp code

Reply:
so the html buttons with chakra don't even look like buttons for some odd reason. I asked brandon to simply change the buttons to the chakra buttons so that the testing ux wasn't completely unusable :/